faculty/clients/model.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2018-2019 Faculty Science Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from attr import attrs, attrib
```

Comment:
I'd prefer we used the "fun" `attr.s` and `attr.ib` functions, but I see there is already a precedent for this in platform context...

Reply:
I was a bit on the fence about this, but their sarcastic 'serious business aliases' comment pushed me over the edge to the other side.

I did come across in the tests though that it's useful to have the `attr` module imported so you can use `attr.evolve`. Maybe we can change this in a future PR? I'll create an issue to essentially poll anyone who might care.